config-model/src/test/java/com/yahoo/vespa/model/content/ContentClusterTest.java

```diff
@@ -1456,25 +1456,32 @@ private void assertIndexingDocprocEnabled(boolean indexed, boolean force, boolea
                 "</services>";
         VespaModel model = createEnd2EndOneNode(new TestProperties(), services);
         var searchCluster = model.getContentClusters().get("search").getSearch();
-        assertEquals(expEnabled, searchCluster.getIndexingDocproc().isPresent());
+        if (force) {
+            assertEquals("default", searchCluster.getIndexingDocproc().getClusterName("search"));
```

Comment:
Do we need to handle the tests on lines 1459-1466 differently? They all seem the same.

Reply:
Simplified